document-store/src/main/java/org/hypertrace/core/documentstore/model/subdoc/UpdateOperator.java

```diff
@@ -0,0 +1,9 @@
+package org.hypertrace.core.documentstore.model.subdoc;
+
+public enum UpdateOperator {
+  SET,
+  UNSET,
```

Comment:
What does `UNSET` mean? Does it remove the full sub-document path?

Reply:
Yes.